test/js/ez-browse.js

```diff
@@ -0,0 +1,95 @@
+describe('ez-browse', function() {
+    let element;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-browse'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectedLocationId`', function () {
+            it('should be set by the attribute', function () {
+                const value = '42';
+
+                element.setAttribute('selected-location-id', value);
+                assert.equal(element.selectedLocationId, value);
+            });
+        });
+    });
+
+    describe('content discover', function () {
+        function simulateClick(element) {
+            const click = new CustomEvent('click', {
+                bubbles: true,
+                cancelable: true,
+            });
+
+            element.dispatchEvent(click);
+            return click;
+        }
+
+        it('should dispatch contentDiscover with config and listeners on click', function () {
+            element.addEventListener('ez:contentDiscover', (e) => {
+                assert.equal(e.detail.config.title, 'Browse', 'should provid the title');
+                assert.equal(e.detail.config.startingLocationId, 42, 'should provid the startingLoactionId');
+                assert.equal(e.detail.config.confirmLabel, 'View this content', 'should provid the confirmLabel');
+                assert.isFunction(e.detail.listeners['ez:confirm'], 'should provid `ez:confirm` eventListener');
+                assert.isTrue(e.bubbles, 'event should bubble');
```

Comment:
actually to check that, you should listener to the event on the `document` for instance (and I would do that in a different test so that there is a test for the event *data* and a test for the event *behaviour*)